.github/workflows/lint.yaml

```diff
@@ -18,12 +18,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Shall we update all checkout action in all workflows?

Reply:
Yea, but it's up to you. These two versions are compatible.